WordPress/src/test/java/org/wordpress/android/models/NoticonUtilsTest.kt

```diff
@@ -0,0 +1,110 @@
+package org.wordpress.android.models
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.R
+
+@RunWith(MockitoJUnitRunner::class)
+class NoticonUtilsTest {
+    private val noteUtils = NoticonUtils()
+
+    @Test
+    fun `transforms mention noticon`() {
+        val gridicon = noteUtils.noticonToGridicon("\uf814")
+
+        assertThat(gridicon).isEqualTo(R.drawable.ic_mention_white_24dp)
+    }
```

Comment:
ehm...just wanted to try the multiline comment new feature 😊...and say I like you added unit tests 😎!